stix_shifter_modules/msatp/configuration/lang_en.json

```diff
@@ -20,6 +20,24 @@
         "sni": {
             "label": "Server Name Indicator",
             "description": "The Server Name Indicator (SNI) enables a separate hostname to be provided for SSL authentication"
+        },
+        "options": {
+            "includeAlerts": {
+                "label": "Join events with Alerts?",
+                "description": "With this feature enabled every event will be searched for correlated alerts in the DeviceAlertsEvent table and the alert data will be included in the results. Keep in mind that this feature will add API CPU consumption."
+            },
+            "includeHostOs": {
+                "label": "Join events with Host OS info?",
+                "description": "With this feature enabled every event will include also the device OS details from the DeviceInfo table. Keep in mind that this feature will add API CPU consumption."
+            },
+            "includeNetworkInfo": {
+                "label": "Join events with Device Network info?",
+                "description": "With this feature enabled every event will include also the active network adapters IP and MAC addresses of the device at the time of the event from the DeviceNetworkInfo table. Keep in mind that this feature will add API CPU consumption."
+            },
+            "retainOriginal": {
+                "label": "Keep the original json payload before translation base64 encoded as original_ref?",
```

Comment:
can you please make a shorter label? this would look really odd in the UI and it would be harder to read. Description already contain the details of the field.

Reply:
done